web/src/engine/websites/DynastyScans_e2e.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
+    plugin: {
+        id: 'dynasty-scans',
+        title: 'DynastyScans'
+    },
+    container: {
+        url: 'https://dynasty-scans.com/series/175160', //One Piece

```

Comment:
The comment is incorrect, could be removed anyway as the answer is just a click away

Reply:
oops; copy paste